src/modules/common/components/TextButton.js

```diff
@@ -15,13 +15,14 @@ export const StyledButton = styled.button`
 
 type ButtonPropsType = {
   onClick: () => void,
-  text: string
+  text: string,
+  disabled: boolean
```

Comment:
There should be other usages of the `TextButton`. So you probably either have to make the prop optional or supply the prop there too. Otherwise flow should show errors. Do you already have flow configured?